src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && (<p className="Person__age">{`I am ${age}`}</p>) }
+
+      <p className="Person__partner">
+        { isMarried
+          ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
```

Comment:
```suggestion
          ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
```

I guess you can move it to separate variable like partner or partnerSex and use it here